src/app/components/Token.tsx

```diff
@@ -50,20 +50,18 @@ export default function Token({
         }}
         className="flex w-full justify-end"
       >
-        <div className="absolute mt-1 mr-2">
-          <div className="flex gap-1">
-            <TokenScoreBadge
```

Comment:
we probably still want that for gacha information ^^

Reply:
It only work on localhost actually...

Reply:
wait.. i saw it works well in production now..
<img width="1455" alt="Screenshot 2024-09-22 at 9 05 52 AM" src="https://github.com/user-attachments/assets/81cdcf68-001b-4d4d-99a7-18b8f9553eae">

Reply:
wtf, okay, that's really nice!

Reply:
bring it back!!